command/agent/metrics_endpoint.go

```diff
@@ -22,6 +22,12 @@ func (s *HTTPServer) MetricsRequest(resp http.ResponseWriter, req *http.Request)
 	}
 
 	if format := req.URL.Query().Get("format"); format == "prometheus" {
+
+		// Only return Prometheus formatted metrics if the user has enabled
+		// this functionality. GH-10070.
```

Comment:
Usually we only leave GitHub issue IDs in if we _haven't_ fixed something. For something we've already fixed, the commit log will tell us what's up.

```suggestion
		// this functionality.
```